src/scanpy/tools/_umap.py

```diff
@@ -56,6 +56,7 @@ def umap(
     key_added: str | None = None,
     neighbors_key: str = "neighbors",
     copy: bool = False,
+    parallel: bool = False,
```

Comment:
I think we have the `n_jobs: int | None` convention for this (with `None` meaning `sc.settings.N_JOBS`), but `simplicial_set_embedding` just passes `parallel` on to numba.

We should think about how the two integrate before we add this.

Reply:
Yes, that’s why we should talk about the parameter name. IIRC that would be the first parallelization parameter not called `n_jobs`.